contents/teams/replay/objectives.mdx

```diff
@@ -1,17 +1,19 @@
-### 🎯 Goal 1: 10 Happy Android Replay Users 
-
-We'll also release an alpha of iOS Mobile Replay.
+### 🎯 Goal 1: Quality and Consolidation
 
 Why? 
 
-We've done the work's base and now we want to release the new Android SDK with session replay and onboard customers to validate the product. 
+We've done a lot of investigation and preparation work and we want to captilise on that
+
+* Mobile replay out of beta for Android/iOS
+* Build the best UX for capturing and playing recordings
+* Build the best UX for finding recordings
 
-### 🎯 Goal 2: People love replay even more
+### 🎯 Goal 2: Error tracking MVP
 
-Why? - we want this product to be fit for everyone and understand where it doesn't spark joy and how we can increase the value people receive and feel.
+Why? PostHog is "The single platform to analyze, test, observe, and deploy new features". Having network timings, network payloads, and console errors in Session Replay is a great developer tool. We want to learn by doing to understand how we could capture and expose even more information here to help people dig in to what is happening with their products
 
-### 🎯 Goal 3: First-class mobile support in app.posthog.com
+Follow along in [our Q2 MegaIssue](https://github.com/PostHog/posthog/issues/21078)
```

Comment:
I'd move this up or down (before goal 1 or after goal 3) otherwise it feels only related to the goal 2

Reply:
Lazy editing 🙈